src/components/Bullet.tsx

```diff
@@ -531,16 +532,15 @@ const Bullet = ({
   // expand or collapse on click
   // has some additional logic to make it work intuitively with pin true/false
   const clickHandler = useCallback(
-    (e: React.MouseEvent) => {
-      // stop click event from bubbling up to Content.clickOnEmptySpace
-      e.stopPropagation()
+    (e: FastClickEvent) => {
       // short circuit if dragHold
       // useLongPress stop is activated in onMouseUp but is delayed to ensure that dragHold is still true here
       // stopping propagation from useLongPress was not working either due to bubbling order or mismatched event type
       if (dragHold) return
 
       // short circuit if toggling multiselect
-      if (!isTouch && (isMac ? e.metaKey : e.ctrlKey)) {
+      // e should always be a MouseEvent if !isTouch, but getting the type system to believe it is another story
+      if (!isTouch && e instanceof MouseEvent && (isMac ? e.metaKey : e.ctrlKey)) {
```

Comment:
Just checking, why do we need to convince the type system that `e` is a `MouseEvent`? I'm not seeing any issues with the following, but I may be missing something:

```suggestion
      if (!isTouch && (isMac ? e.metaKey : e.ctrlKey)) {
```

Reply:
It looks OK. I tried a couple of different definitions for `FastClickEvent` so this one may not need the type guard, as long as it passes CI.